ee/tabby-webserver/src/schema/mod.rs

```diff
@@ -419,6 +419,21 @@ impl Mutation {
         Ok(true)
     }
 
+    async fn upload_user_avatar_base64(
+        ctx: &Context,
+        id: ID,
+        avatar: Option<String>,
+    ) -> Result<bool> {
+        let claims = check_claims(ctx)?;
+        if claims.sub.0 != id && check_admin(ctx).is_err() {
```

Comment:
this doesn't compile. Please always test your PR locally before marking it ready for review.

Reply:
Seems this was an issue of rebasing on main, though I don't know why the compiler error was showing here and not for me locally until I rebased